### ISSUE TITLE
ComboBox input is aria-hidden while popover is open

### ISSUE THREAD

**Issue by devongovett** [adobe/react-spectrum#1093]
# 🐛 Bug Report 

When a ComboBox is open, the menu popover is considered "modal", which means that aria-hidden is added to all surrounding content. Unfortunately, this also includes the combo box input itself. This means the user cannot navigate in and out of the menu. We don't want to disable aria-hidden completely because with portals there may be content between the input and menu since the menu is mounted at the end of the document. We need to investigate how we can make everything *except* the input and menu hidden instead, so the user can navigate only between the menu and input and not to other surrounding content.

## 🧢 Your Company/Team

RSP
